### PR TITLE
Rename: elements -> core-elements

### PR DESCRIPTION
**Pull request by manolo**
Connected to vaadin/components-team-tasks#50

[Review on Reviewable](https://reviewable.io/reviews/vaadin/vaadin-elements/27)

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## Vaadin Core Elements 0.3.0-beta... (...)
+- Added Core word, so as the bundle includes all the set of Vaadin Components licensed as Apache 2.0.
```
Comment (jounik) [inline CHANGES.md]: Needs rewording, should not say “Vaadin Components” for instance.

Here’s my suggestion:
The product and related releases are renamed to “Vaadin Core Elements”, which is the collection of custom elements (i.e. Web Components) that are licensed under the Apache License 2.0. The term “Vaadin Elements” refers to all custom elements created by Vaadin from now on, including commercial elements.

Comment (jounik) [inline CHANGES.md]: Also, why do we still have a CHANGES.md file? The same should apply here as with the elements, that we only have the release notes in GitHub.

File: README.md [hunk 2]
```diff
@@ -69,25 +69,25 @@ We offer three ways to use Vaadin Elements in your project: Bower, CDN and ZIP a
 
  > **Serving the files during development**, when using Bower or the ZIP archive:
 
- > Due to browser security restrictions, serving HTML imports from a `file:///` URL does not work. You need a web server to view pages where you use custom elements. One simple option is to use the [`serve`](https://www.npmjs.com/package/serve) NPM package.
+ > Due to browser security restrictions, serving HTML imports from a `file:///` URL does not work. You need a web server to view pages where you use custom  elements. One simple option is to use the [`serve`](https://www.npmjs.com/package/serve) NPM package.
```
Comment (jounik) [inline README.md]: Unnecessary whitespace change.

**Comment by tomivirkki** [comment]
Review status: 0 of 10 files reviewed at latest revision, 5 unresolved discussions.

---

*[README.md, line 14 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-elements/27#-K5VpenR9KE7Vyt-lnkX:-K5VpenR9KE7Vyt-lnkY:1517115608) ([raw file](https://github.com/vaadin/vaadin-elements/blob/7896943647f5c182acc2adcde014cf2ed015d2d8/README.md#L14)):*
Combo-box examples url has /vaadin-elements/

---

*[README.md, line 56 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-elements/27#-K5VpzuEjVEPoTffPu6K:-K5VpzuEjVEPoTffPu6L:34238599) ([raw file](https://github.com/vaadin/vaadin-elements/blob/7896943647f5c182acc2adcde014cf2ed015d2d8/README.md#L56)):*
says vaadin-elements in the url

---

*[README.md, line 72 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-elements/27#-K5VjmkqNSNaW92r-cb3:-K5VjmkqNSNaW92r-cb4:381876913) ([raw file](https://github.com/vaadin/vaadin-elements/blob/7896943647f5c182acc2adcde014cf2ed015d2d8/README.md#L72)):*
Extra space

---

*[README.md, line 82 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-elements/27#-K5Vq64oNA4d1ATkusoa:-K5Vq64oNA4d1ATkusob:1693542503) ([raw file](https://github.com/vaadin/vaadin-elements/blob/7896943647f5c182acc2adcde014cf2ed015d2d8/README.md#L82)):*
Says vaadin-elements in the url

---

*[tasks/cdn.js, line 30 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-elements/27#-K5VqDzuWUsgDeRqPzYN:-K5VqDzuWUsgDeRqPzYO:-736142878) ([raw file](https://github.com/vaadin/vaadin-elements/blob/7896943647f5c182acc2adcde014cf2ed015d2d8/tasks/cdn.js#L30)):*
core-vaadin-elements -> vaadin-core-elements

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-elements/27)*

**Comment by tomivirkki** [comment]
Also the apidoc/index.html file has vaadin-elements in it

---







*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-elements/27#-:-K5VqzU9lIj3BjGIzFBr:1053375320)*

**Comment by manolo** [comment]
Review status: 0 of 10 files reviewed at latest revision, 7 unresolved discussions.

---

*[CHANGES.md, line 2 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-elements/27#-K5VfvotN7IkCXRagNHl-r1-2:-K5ZMCvLXRsGNT7W9koG:-669477229) ([raw file](https://github.com/vaadin/vaadin-elements/blob/7896943647f5c182acc2adcde014cf2ed015d2d8/CHANGES.md#L2)):*
Changed the wording, I think we should remove this file in a separate PR when we make the new release.

---

*[README.md, line 14 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-elements/27#-K5VpenR9KE7Vyt-lnkX:-K5ZMq886E4xfeyBPO3F:-499146009) ([raw file](https://github.com/vaadin/vaadin-elements/blob/7896943647f5c182acc2adcde014cf2ed015d2d8/README.md#L14)):*
Done.

---

*[README.md, line 56 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-elements/27#-K5VpzuEjVEPoTffPu6K:-K5ZMv_t85GEUb-_PnT1:-499146009) ([raw file](https://github.com/vaadin/vaadin-elements/blob/7896943647f5c182acc2adcde014cf2ed015d2d8/README.md#L56)):*
Done.

---

*[README.md, line 72 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-elements/27#-K5VfvotN7IkCXRagNHm-r1-72:-K5ZNOv-zxKsz12qQgar:-499146009) ([raw file](https://github.com/vaadin/vaadin-elements/blob/7896943647f5c182acc2adcde014cf2ed015d2d8/README.md#L72)):*
Done.

---

*[README.md, line 72 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-elements/27#-K5VjmkqNSNaW92r-cb3:-K5ZNPJD8eO73jK3kUQL:-499146009) ([raw file](https://github.com/vaadin/vaadin-elements/blob/7896943647f5c182acc2adcde014cf2ed015d2d8/README.md#L72)):*
Done.

---

*[README.md, line 82 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-elements/27#-K5Vq64oNA4d1ATkusoa:-K5ZNRFEt1WI9Byax5t1:-499146009) ([raw file](https://github.com/vaadin/vaadin-elements/blob/7896943647f5c182acc2adcde014cf2ed015d2d8/README.md#L82)):*
Done.

---

*[tasks/cdn.js, line 30 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-elements/27#-K5VqDzuWUsgDeRqPzYN:-K5ZNfANCI__hnOz4iVP:-499146009) ([raw file](https://github.com/vaadin/vaadin-elements/blob/7896943647f5c182acc2adcde014cf2ed015d2d8/tasks/cdn.js#L30)):*
Done.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-elements/27)*

**Comment by tomivirkki** [comment]
Review status: 0 of 11 files reviewed at latest revision, 7 unresolved discussions.

---

*[README.md, line 56 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-elements/27#-K5VpzuEjVEPoTffPu6K:-K5ZYC-SUnCQFlD4GbK5:-445653408) ([raw file](https://github.com/vaadin/vaadin-elements/blob/7896943647f5c182acc2adcde014cf2ed015d2d8/README.md#L56)):*
The url is now ok but the file itself (vaadin-core-elements.html) doesn't exist

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-elements/27)*

**Comment by Saulis** [comment]
As discussed in Skype, this PR will be targeted to 0.3.0 instead of master branch.

---

Review status: 0 of 11 files reviewed at latest revision, 7 unresolved discussions.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-elements/27#-:-K5ZZMll0_CJ4Iu0xW9C:-38479209)*

**Comment by manolo** [comment]
Review status: 0 of 12 files reviewed at latest revision, 7 unresolved discussions.

---

*[README.md, line 56 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-elements/27#-K5VpzuEjVEPoTffPu6K:-K5ZicaXvt69NbYPtXzt:-870787312) ([raw file](https://github.com/vaadin/vaadin-elements/blob/7896943647f5c182acc2adcde014cf2ed015d2d8/README.md#L56)):*
Done. Yep forgot the git add step

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-elements/27)*

**Comment by manolo** [comment]
Closing this PR. Conversation continues in #28
